ui/admin/app/routes/scopes/scope/sessions/index.js

```diff
@@ -4,5 +4,172 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
 
-export default class ScopesScopeSessionsIndexRoute extends Route {}
+export default class ScopesScopeSessionsIndexRoute extends Route {
+  // =services
+
+  @service store;
+
+  // =attributes
+
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+    users: {
+      refreshModel: true,
+      replace: true,
+    },
+    targets: {
+      refreshModel: true,
+      replace: true,
+    },
+    status: {
+      refreshModel: true,
+      replace: true,
+    },
+    page: {
+      refreshModel: true,
+    },
+    pageSize: {
+      refreshModel: true,
+    },
+  };
+
+  allSessions;
+  associatedUsers;
+  associatedTargets;
+
+  /**
+   * Loads all sessions under the current scope and encapsulates them into
+   * an array of objects containing their associated users and targets.
+   * @return {Promise{[{sessions: [SessionModel], allSessions: [SessionModel], associatedUsers: [UserModel], associatedTargets: [TargetModel], totalItems: number}]}}
+   */
+  async model({ search, users, targets, status, page, pageSize }) {
+    const { id: scope_id } = this.modelFor('scopes.scope');
+    const filters = {
+      scope_id: [{ equals: scope_id }],
+      status: [],
+      user_id: [],
+      target_id: [],
+    };
+    users.forEach((user) => {
+      filters.user_id.push({ equals: user });
+    });
+    targets.forEach((target) => {
+      filters.target_id.push({ equals: target });
+    });
+    status.forEach((item) => {
+      filters.status.push({ equals: item });
+    });
+
+    const queryOptions = {
+      scope_id,
+      recursive: true,
```

Comment:
I think we should also remove these scope id and recursive parameters from the store queries in this file since they're not doing anything right now. We can revisit not having it hard coded later

Reply:
Yep. I noticed that after the change in the groups work. Removing now.